src/main/java/com/google/jenkins/plugins/computeengine/ComputeEngineWindowsLauncher.java

```diff
@@ -114,7 +114,7 @@ protected void launch(ComputeEngineComputer computer, TaskListener listener, Ins
             SCPClient scp = conn.createSCPClient();
 
             logInfo(computer, listener, "Copying slave.jar to: " + TMPDIR);
-            scp.put(Jenkins.getInstance().getJnlpJars("slave.jar").readFully(), "slave.jar", TMPDIR);
+            scp.put(Jenkins.get().getJnlpJars("slave.jar").readFully(), "slave.jar", TMPDIR);
```

Comment:
Following up on the other thread, it would be good to see this as a variable or static constant instead of hard-coded.

Reply:
Yea, that would require some more digging.